lightning/src/ln/channel.rs

```diff
@@ -1865,13 +1865,12 @@ impl<ChanSigner: ChannelKeys> Channel<ChanSigner> {
 
 			// +1 for this HTLC.
 			let local_commit_tx_fee_msat = self.next_local_commit_tx_fee_msat(1);
-			if self.value_to_self_msat < self.local_channel_reserve_satoshis * 1000 + local_commit_tx_fee_msat {
-				return Err(ChannelError::Close("Cannot receive value that would put us under local channel reserve value".to_owned()));
+			if self.value_to_self_msat < self.channel_reserve_satoshis * 1000 + local_commit_tx_fee_msat {
+				return Err(ChannelError::Close("Cannot receive value that would put us under our channel reserve value".to_owned()));
```

Comment:
The side ("our") of the reserve is immaterial here since its implied by this being a receive. I'd try to avoid "us" and "our" in these messages by stating as:

> Cannot receive value that would put the channel under its reserve value

That said, there are other messages that use "our" and "their" pronouns it appears.

Reply:
Good suggestion but I took it a bit differently "Cannot accept HTLC that would put counterparty balance under channel reserve value" as it underscores both purpose of this check and ownership of checked value.